requirements.txt

```diff
@@ -12,3 +12,4 @@ psutil
 pylatexenc >= 2.0, < 3.0
 scipy >= 1.7.0, < 2.0.0
 termcolor
+scikit-image
```

Comment:
Probably not terribly hard but it might be slow, scikit implements the contour find part in cython. I could try to do myself if this is a sticking point but for now it works so I'll focus on fixing the other comments